tensorflow_serving/batching/batching_util_test.cc

```diff
@@ -0,0 +1,93 @@
+/* Copyright 2017 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow_serving/batching/batching_util.h"
+
+
+
+#include <gtest/gtest.h>
+#include "tensorflow/core/framework/tensor.h"
+#include "tensorflow/core/lib/core/status.h"
+#include "tensorflow/core/platform/types.h"
+
+namespace tensorflow {
+namespace serving {
+namespace {
+std::vector<std::pair<string, Tensor>> CreateInputsWithTensorShapes(
```

Comment:
Please add a comment explaining what this function does, and the format of the output.

(Please go through the entire PR and make sure all functions/methods are documented either in the .h file, or in the .cc file for ones not part of a .h file like this one. Thanks!)